src/world_model/nav2_costmap_world_model/CMakeLists.txt

```diff
@@ -12,13 +12,9 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
-find_package(geometry_msgs REQUIRED)
-find_package(std_msgs REQUIRED)
-find_package(nav2_tasks)
-find_package(nav2_util)
-find_package(nav2_libs_msgs)
-find_package(nav2_world_model_msgs)
-find_package(nav_msgs)
+find_package(nav2_libs_msgs REQUIRED)
```

Comment:
After PR #107 I think all the messages are now in nav2_msgs, so these need to be changed.